deploy

```diff
@@ -1,10 +1,19 @@
 #!/bin/bash
+set -ex
 
-# script to deploy software. It is called by the deploy service or can be run
-# manually.
-LOG=~/deploy_lwa110-pyutils.log
-DATE="`date --iso-8601='seconds'`"
-echo "$DATE: `which python`" > $LOG
-output=`pip install .`
```

Comment:
check $? after each command for errors and log if necessary. exit if necessary with same error number.